src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -154,104 +154,30 @@ functional_modules::ProcessResult VRModule::HandleMessage(
   return ProcessResult::PROCESSED;
 }
 
-functional_modules::ProcessResult VRModule::HandleHMIMessage(
-  application_manager::MessagePtr msg) {
+functional_modules::ProcessResult VRModule::ProcessHMIMessage(
+    application_manager::MessagePtr msg) {
   LOG4CXX_AUTO_TRACE(logger_);
-
-  if (!SetHMIMessageType(msg)) {
-    return ProcessResult::FAILED;
-  }
-  msg->set_protocol_version(application_manager::ProtocolVersion::kV3);
-
-  switch (msg->type()) {
-    case application_manager::MessageType::kNotification: {
-      if (functional_modules::hmi_api::on_service_deactivated
-          == msg->function_name()) {
-          commands::OnServiceDeactivatedNotification notification(this);
-          notification.Execute(msg);
-         // TODO(giang): Un-comment when OnDefaultServiceChosen
-         // notification was implemented
-         // } else if (functional_modules::hmi_api::on_default_service_chosen
-         // == function_name) {
-         //  commands::OnDefaultServiceChosen notification(msg);
-         //  notification.Execute();
-         //  }
-      }
-      break;
-    }
-    case application_manager::MessageType::kRequest: {
-      commands::Command* command = MobileCommandFactory::CreateCommand(this, msg);
-      if (command) {
-        request_controller_.AddRequest(msg->correlation_id(), command);
-        command->Run();
-      } else {
-        return ProcessResult::CANNOT_PROCESS;
-      }
-      break;
-    }
-    default: {
-      return ProcessResult::FAILED;
-    }
-  }
   return ProcessResult::PROCESSED;
```

Comment:
As it's not processed please return appropriate value. And please add comment that we don't handle json between plugin and hmi anymore